spec/features/view_album_spec.rb

```diff
@@ -18,10 +18,10 @@
 
     expect(page).to have_content album.description
     within('#photo0') do
-      expect(find('img')['src']).to have_content "ice-boats.jpg"
+      expect(find('img')['src']).to have_content "Airturbine.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.